package.json

```diff
@@ -34,62 +34,63 @@
     "prettify:package": "yarn prettier-package-json --write **/package.json"
   },
   "dependencies": {
-    "@babel/core": "^7.3.3",
-    "@storybook/addon-a11y": "^5.0.1",
-    "@storybook/addon-actions": "^5.0.1",
-    "@storybook/addon-info": "^5.0.1",
-    "@storybook/addon-knobs": "^5.0.1",
-    "@storybook/addon-links": "^5.0.1",
-    "@storybook/addon-storysource": "^5.0.1",
-    "@storybook/addons": "^5.0.1",
-    "@storybook/react": "^5.0.1",
+    "@babel/core": "^7.4.0",
+    "@storybook/addon-a11y": "^5.0.5",
+    "@storybook/addon-actions": "^5.0.5",
+    "@storybook/addon-info": "^5.0.5",
+    "@storybook/addon-knobs": "^5.0.5",
+    "@storybook/addon-links": "^5.0.5",
+    "@storybook/addon-storysource": "^5.0.5",
+    "@storybook/addons": "^5.0.5",
+    "@storybook/react": "^5.0.5",
     "@storybook/storybook-deployer": "^2.8.1",
     "@svgr/webpack": "^4.1.0",
-    "@types/enzyme": "^3.1.18",
+    "@types/enzyme": "^3.9.1",
     "@types/enzyme-adapter-react-16": "^1.0.5",
     "@types/jest": "^24.0.11",
-    "@types/node": "^11.11.1",
-    "@types/react": "^16.8.7",
-    "@types/react-dom": "^16.8.2",
+    "@types/node": "^11.12.0",
+    "@types/react": "^16.8.8",
+    "@types/react-dom": "^16.8.3",
```

Comment:
Shouldn't both `react` and `react-dom` have the same version?

Reply:
This was added by merging with https://github.com/Amsterdam/amsterdam-styled-components/pull/7

Reply:
I understand, but shouldn't all versions of `react`, `react-dom`, `@types/react` and `@types/react-dom` be in sync?

Reply:
Yes, true